src/HttpKernel/InstallationKernel.php

```diff
@@ -67,7 +67,6 @@ public function canBootRealSystem()
     {
         return file_exists($this->getRootDir().'/config/parameters.yml')
             && file_exists($this->getRootDir().'/../system/config/localconfig.php')
```

Comment:
Hm, I'm pretty sure that removing this would cause problems. Also, in a regular installation there will always be a `localconfig.php` file, so …